source/features/softwrap-code.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+Soft-wrap code inside codeblocks
+*/
+import './softwrap-code.css';
+import select from 'select-dom';
+import features from '../libs/features';
+
+async function init(): Promise<void> {
+	// Classes:
+	// `.diff-table` - Split and unified diffs
+	// `.d-table` - Suggested changes
+	// `.js-file-line-container` - Embedded code blocks
+	const tables = select.all('.diff-table, .d-table, .js-file-line-container');
+
+	for (const table of tables) {
+		if (table.classList.contains('rgh-softwrapped-code')) {
+			continue;
+		}
+
+		table.classList.add('rgh-softwrapped-code');
+
+		const lines = select.all('.blob-code-inner', table);
+		for (const line of lines) {
+			if (!line.firstChild || !line.firstChild.textContent) {
+				continue;
+			}
+
+			const leadingSpaceCharacters = line.firstChild.textContent.match(/^\s+/);
```

Comment:
```suggestion
			const leadingSpaceCharacters = line.firstChild!.textContent!.match(/^\s+/);
```

This lets you skip the previous `if`